smash-shells.lic

```diff
@@ -66,12 +66,11 @@ loop do
   end
 
   waitrt?
-  DRCT.walk_to(16_236)
   next unless Flags['shell-fragments']
   pause 4
   DRC.wait_for_script_to_complete('tendme')
-  pause 20
-  DRC.wait_for_script_to_complete('tendme')
+  #pause 20
+  #DRC.wait_for_script_to_complete('tendme')
```

Comment:
Why was this commented out?

Reply:
Tendme was fixed to get all lodged and bleeders in one go. I left in just in case it was needed but I have run it since the beginning of HE and have not had a problem, other than getting killed by an otter horde because I dropped the fragments in the room.

Reply:
Cool, thanks